pages/apim/3.x/user-guide/publisher/design-studio/design-studio-debug-mode.adoc

```diff
@@ -0,0 +1,111 @@
+= Debug Mode
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_publisherguide_design_studio_debug_mode.html
+:page-folder: apim/user-guide/publisher/design-studio
+:page-layout: apim3x
+
+
+[label label-version]#New in version 3.16#
+
+== Overview
+
+From 3.16, we've updated the try it mode to offer new capabilities and view what's happening on each policy executed, now called Debug mode.
+
+Debug mode is a tool for troubleshooting your API proxies running on APIM. It provides detailed information about the behavior of each policy in your flows as well as the overall execution order at runtime. With debug, you can:
+
+* Understand which policies are triggered by a specific request (including Platform-level policies).
+* Visualise which policies are executed or skipped (conditional policy).
+* Understand the order of execution of your policies.
+* Trace transformations and changes made to headers, body and execution context.
+* Easily troubleshoot and find root cause of errors, misbehaviors or performance issues.
+
+== How to Get Started With Debug Mode
+
+. link:{{ '/apim/3.x/apim_publisherguide_design_studio_create.html' | relative_url }}[Design your API^].
+. Select *Debug* tab.
+. Configure your request.
+. Select *Send*.
+
+== How To Test Your API Concept with Debug Mode
+
+You can send a request to your *API* endpoint through the embedded HTTP client available under the *Debug* tab.
+
+NOTE: You don't need to save or redeploy the API to test your changes when using the Debug Mode.
+
+image:{% link images/apim/3.x/api-publisher-guide/design-studio/debug-mode/debug-mode-example-empty.png %}[Debug Mode empty example]
+
+The embedded HTTP client let's you configure different attributes of your request:
+
+* *HTTP method* to select the HTTP verb (e.g., `GET`, `POST`, `PUT`, etc.).
+* *Path* to test a specific subpath and provide query parameters. 
+
+NOTE: The context-path of the API your are debugging is already taken into account. Path refers to the subpath: myapi-context-path/*path*?queryparam=value
+
+* *Headers* to provide a custom list of HTTP request headers.
+* *Request body* to configure a custom request body. 
+
+Since debug mode emulates an external call to your API, it will resolve the flow as the gateway would do, so if you want to debug a specific flow you must configure your request (path, headers) to replicate the conditions at which the flow is triggered.
```

Comment:
```suggestion
Since debug mode emulates an external call to your API, it will resolve the flow as the gateway would do, so if you want to debug a specific flow you must configure your request (path, headers) to replicate the conditions on which the flow is triggered.
```